### PR TITLE
Bug 1308712 - Add Spark job detail view

### PR DESCRIPTION
**Pull request by maurodoglio**
This also fixes a few bugs in the spark job startup code.

---
This change is [Reviewable](https://reviewable.io/reviews/mozilla/telemetry-analysis-service/35)

### CODE REVIEW

File: atmo/jobs/views.py [hunk 1]
```diff
@@ -2,10 +2,11 @@
 from django.views.decorators.http import require_POST
 from django.contrib.auth.decorators import login_required
 from django.http import HttpResponseBadRequest
-from django.shortcuts import redirect
+from django.shortcuts import redirect, get_object_or_404, render
 
 from session_csrf import anonymous_csrf
 
+from atmo.jobs.models import SparkJob
```
Comment (jezdez) [inline atmo/jobs/views.py]: mind doing `from .models import SparkJob` here?

File: atmo/templates/atmo/dashboard.html [hunk 2]
```diff
@@ -60,7 +60,7 @@
         {% for spark_job in user_spark_jobs %}
         <tr>
           <td class="hidden">{{ spark_job.id }}</td>
-          <td>{{ spark_job.identifier }}</td>
+          <td><a href="{% url "jobs-detail" id=spark_job.id %}">{{ spark_job.identifier }}</a></td>
```
Comment (jezdez) [inline atmo/templates/atmo/dashboard.html]: It'd be cool to define a `get_absolute_url` in the `SparkJob` and use that instead

File: atmo/templates/atmo/detail-spark-job.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
+{% extends "atmo/base.html" %}
+{% block content %}
+<h1>Spark Job {{ job.identifier }}</h1>
+<p class="lead">Summary:</p>
+<div class="row">
+  <div class="col-md-2"><strong>Notebook S3 key:</strong></div>
+  <div class="col-md-10">{{ job.notebook_s3_key}}</div>
```
Comment (jezdez) [inline atmo/templates/atmo/detail-spark-job.html]: whitespace before the `}}` please, multiple instances of that in this file

File: atmo/utils/scheduling.py [hunk 4]
```diff
@@ -72,7 +72,7 @@ def spark_job_run(user_email, identifier, notebook_uri, result_is_public, size,
                 'Path': 's3://{}/bootstrap/telemetry.sh'.format(
                     settings.AWS_CONFIG['SPARK_EMR_BUCKET']
                 ),
-                'Args': ['--timeout', job_timeout]
+                'Args': ['--timeout', str(job_timeout)]
```
Comment (jezdez) [inline atmo/utils/scheduling.py]: 👍

File: atmo/templates/atmo/detail-spark-job.html [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  <div class="col-md-2"><strong>Interval in hours:</strong></div>
+  <div class="col-md-10">{{ job.interval_in_hours}}</div>
+</div>
+<div class="row">
+  <div class="col-md-2"><strong>Timeout:</strong></div>
+  <div class="col-md-10">{{ job.job_timeout}}</div>
+</div>
+<div class="row">
+  <div class="col-md-2"><strong>Start date:</strong></div>
+  <div class="col-md-10">{{ job.start_date}}</div>
```
Comment (jezdez) [inline atmo/templates/atmo/detail-spark-job.html]: Should we format the start and end dates somehow nicely?

File: atmo/templates/atmo/detail-spark-job.html [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+</div>
+<div class="row">
+  <div class="col-md-2"><strong>End date:</strong></div>
+  <div class="col-md-10">{{ job.end_date}}</div>
+</div>
+<div class="row">
+  <div class="col-md-2"><strong>Is enabled:</strong></div>
+  <div class="col-md-10">{{ job.is_enabled}}</div>
```
Comment (jezdez) [inline atmo/templates/atmo/detail-spark-job.html]: Maybe how a checkmark glyph here if `is_enabled` is `True`?

File: atmo/templates/atmo/detail-spark-job.html [hunk 7]
```diff
@@ -0,0 +1,43 @@
+{% extends "atmo/base.html" %}
+{% block content %}
```
Comment (jezdez) [inline atmo/templates/atmo/detail-spark-job.html]: Mind providing a `head_title` block as well? it'll be used in the `<title>` element

File: atmo/jobs/models.py [hunk 8]
```diff
@@ -134,10 +134,9 @@ def delete(self, *args, **kwargs):
     @classmethod
     def step_all(cls):
         """Run all the scheduled tasks that are supposed to run."""
-        now = datetime.now()
```
Comment (jezdez) [inline atmo/jobs/models.py]: the import needs to be removed, too

File: atmo/jobs/views.py (hunk 1 above)
Comment (maurodoglio) [inline atmo/jobs/views.py]: 👍

File: atmo/templates/atmo/dashboard.html (hunk 2 above)
Comment (maurodoglio) [inline atmo/templates/atmo/dashboard.html]: 👍

File: atmo/templates/atmo/detail-spark-job.html (hunk 3 above)
Comment (maurodoglio) [inline atmo/templates/atmo/detail-spark-job.html]: 👍

File: atmo/templates/atmo/detail-spark-job.html (hunk 5 above)
Comment (maurodoglio) [inline atmo/templates/atmo/detail-spark-job.html]: 👍

File: atmo/templates/atmo/detail-spark-job.html (hunk 6 above)
Comment (maurodoglio) [inline atmo/templates/atmo/detail-spark-job.html]: 👍

File: atmo/jobs/models.py (hunk 8 above)
Comment (maurodoglio) [inline atmo/jobs/models.py]: 👍

**Pull request merged**